swap_meet/item.py

```diff
@@ -1,2 +1,37 @@
 class Item:
-    pass
\ No newline at end of file
+
+    def __init__(self, condition = 0.0, age = 0.0,category= None):
+        if category == None:
+            category = ""
+        self.category = category
+        if condition == 0.0:
+            condition = 0.0
+        self.condition = condition
+        if age == 0.0:
+            age = 0.0
+        self.age = age
```

Comment:
You only need to do a check on the default values for things mutable data types.  

Everything else you can just set as a default parameter:

```suggestion
    def __init__(self, condition=0.0, age=0.0, category=""):
        self.category = category
        self.condition = condition
        self.age = age
```

Reply:
Glad to hear you are feeling better. Thanks for your comments and suggestions. I should have thought about that. Good catch :)